docs/content/2.examples/1.arbitrary-parser.md

```diff
@@ -0,0 +1,44 @@
+# Arbitrary parser
```

Comment:
let's also call this folder `recipes` so that we're consistent with the other projects? Plus: You can takeover some of the parts of the nuxt-auth overview then: https://sidebase.io/nuxt-auth/recipes